webpack.common.js

```diff
@@ -1,6 +1,7 @@
 /* global __dirname */
 'use strict';
 const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
Is this still required when it's also in the unstyled webpack file?

Reply:
Good catch, haha... yes I should've removed that